changelog/aliasdecly_func.dd

```diff
@@ -1,23 +1,10 @@
 A new syntax is available to declare an alias to a function type
 
 It's now possible to declare an alias to a function type using the `alias` syntax based on the assignment operator.
-The declaration consists of the return type, followed by parameters and ends with optional (member) function attributes:
+The declaration consists of the return type, followed by parameters and ends with optional (member) function attributes. See
+[AliasDeclarationY](https://dlang.org/spec/grammar.html#AliasDeclarationY) for more details.
```

Comment:
This is Ddoc, no Markdown. You need to use e.g. `LINK2` or other macros.
I'm afraid enabling -preview=markdown won't happen overnight as it probably will result in a few changes.

Reply:
yes but dgileadi also uses markdown in his dd entry.

Reply:
[clearly](https://github.com/dlang/dmd/blob/master/changelog/ddoc_markdown.dd#L3)

Reply:
Have a look at the preview: https://dlang.org/changelog/pending.html#ddoc_markdown

Also, I pointed it out on his PR too: https://github.com/dlang/dmd/pull/9645

Reply:
Yes, this is where i understood that you'd put the new switch in the makefile. Anyway i've replaced with a LINK2 now, no need to bother with this details. It's just that he has to fix his link too.